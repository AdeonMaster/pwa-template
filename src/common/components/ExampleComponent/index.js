import React from 'react';

const ExampleComponent = ({ value }) => <p>Value is: {value}</p>;

export default ExampleComponent;
