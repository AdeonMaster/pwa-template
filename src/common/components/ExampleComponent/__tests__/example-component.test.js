import { shallow } from 'enzyme';
import React from 'react';

import ExampleComponent from '..';

describe('Example Component', () => {
  it('Example Component matches snapshot', () => {
    const component = shallow(<ExampleComponent value="love" />);

    expect(component.getElement()).toMatchSnapshot();
  });
});
