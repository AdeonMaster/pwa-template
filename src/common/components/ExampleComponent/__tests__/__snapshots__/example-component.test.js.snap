// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example Component Example Component matches snapshot 1`] = `
<p>
  Value is: 
  love
</p>
`;
